packages/tools/src/tools/annotation/planarFreehandROITool/closedContourEditLoop.ts

```diff
@@ -398,13 +402,20 @@ function completeClosedContourEdit(element: HTMLDivElement) {
   const { viewport, renderingEngine } = enabledElement;
 
   const { annotation, viewportIdsToRender } = this.commonData;
-  const { fusedCanvasPoints } = this.editData;
+  const { fusedCanvasPoints, prevCanvasPoints } = this.editData;
 
   if (fusedCanvasPoints) {
-    const worldPoints = fusedCanvasPoints.map((canvasPoint) =>
+    const updatedPoints = assertInterpolation(this.configuration)
+      ? getInterpolatedPoints(
```

Comment:
The explanation/doc of each function is already on function itself.
I dont see a point of have a function (with a potential overladed name) just because we want to have a better explanation. I would say this is an unnecessary encapsulation we should avoid.